test/e2e-secondary-network/secondary_network_test.go

```diff
@@ -237,6 +390,16 @@ func testSecondaryNetwork(t *testing.T, networkType string, pods []*testPodInfo)
 			t.Fatalf("Error when pinging between interfaces: %v", err)
 		}
 	})
+	t.Run("testreconcilationAfterAgentRestart", func(t *testing.T) {
+		if err := testData.reconcilationAfterAgentRestart(t); err != nil {
+			t.Fatalf("Error when restarting antrea-agent: %v", err)
+		}
+	})
+	t.Run("testremovePodsAndCheckIP", func(t *testing.T) {
+		if err := testData.removePodsAndCheckIP(); err != nil {
+			t.Fatalf("Error when removing the Pod: %v", err)
+		}
+	})
```

Comment:
from what i can tell, the first test is not actually testing anything, while you assume that the second test (`testremovePodsAndCheckIP`) runs after the first one (`testreconcilationAfterAgentRestart`). You should merge these into a single test case, there is no reason to have 2 if they are not meant to be independent.

Reply:
Agree to merge the two tests too (we should keep existing code in testreconcilationAfterAgentRestart which can check we do not mistakenly remove OVS ports after Agent restarts).

Reply:
merged